mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -467,34 +468,34 @@ private Mailbox doDeleteMailbox(MailboxMapper mailboxMapper, Mailbox mailbox, Ma
     }
 
     @Override
-    public void renameMailbox(MailboxPath from, MailboxPath to, MailboxSession session) throws MailboxException {
+    public List<MailboxRenamedResult> renameMailbox(MailboxPath from, MailboxPath to, MailboxSession session) throws MailboxException {
         LOGGER.debug("renameMailbox {} to {}", from, to);
         MailboxPath sanitizedMailboxPath = to.sanitize(session.getPathDelimiter());
         validateDestinationPath(sanitizedMailboxPath, session);
 
         assertIsOwner(session, from);
         MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
 
-        mapper.execute(Mapper.toTransaction(() -> {
```

Comment:
is it a transaction removal?

Reply:
It is a implementation which return null and we provide a new implementation for it.